dask/dataframe/io/io.py

```diff
@@ -134,8 +142,10 @@ def from_pandas(data, npartitions=None, chunksize=None, sort=True, name=None):
         The number of partitions of the index to create. Note that depending on
         the size and index of the dataframe, the output may have fewer
         partitions than requested.
-    chunksize : int, optional
-        The number of rows per index partition to use.
+    chunksize : int, str, optional
+        If an integer is passed, it means the number of rows per index
+        partition to use. If a string is passed, it represents the number of
```

Comment:
I find it helpful to be explicit about how strings are handled. Say

    If a str is passed, it's parsed with :func:`dask.utils.parse_bytes`.

Then people get a link to that page with all of its examples.

Reply:
Also describe the behavior of leaving it unspecified.